src/main/java/io/strimzi/kafka/bridge/config/BridgeConfig.java

```diff
@@ -55,21 +67,35 @@ public HttpConfig getHttpConfig() {
     }
 
     /**
-     * Loads the entire bridge configuration parameters from a related map
+     * Loads the entire bridge configuration parameters from a related map.
+     * Some validation and default values are also applied.
      *
      * @param map map from which loading configuration parameters
      * @return overall bridge configuration
      */
     public static BridgeConfig fromMap(Map<String, Object> map) {
+        validateAndApplyDefaults(map);
         KafkaConfig kafkaConfig = KafkaConfig.fromMap(map);
         HttpConfig httpConfig = HttpConfig.fromMap(map);
-
         return new BridgeConfig(map.entrySet().stream()
                 .filter(e -> e.getKey().startsWith(BridgeConfig.BRIDGE_CONFIG_PREFIX))
                 .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)),
                 kafkaConfig, httpConfig);
     }
 
+    private static void validateAndApplyDefaults(Map<String, Object> map) {
+        if (map.get(METRICS_TYPE) != null) {
+            // get and validate metrics type
+            MetricsType metricsType = MetricsType.fromString((String) map.get(METRICS_TYPE));
+            // apply default Strimzi Metrics Reporter configurations if not present
+            if (metricsType == MetricsType.STRIMZI_REPORTER) {
+                map.putIfAbsent("kafka.metric.reporters", "io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter");
+                map.putIfAbsent("kafka.prometheus.metrics.reporter.listener.enable", "false");
+                map.putIfAbsent("kafka.prometheus.metrics.reporter.allowlist", ".*");
+            }
```

Comment:
Is this really the best place how to handle it? To inject the configuration? As we will not be able to distinguish between the user-provided and defaults in logs etc. If nothing else, an info/warning message should be used here.